invenio_rdm_records/resources/iiif.py

```diff
@@ -97,6 +99,11 @@ class IIIFResourceConfig(ResourceConfig, ConfiguratorMixin):
                 code=403, description=_("Image size limit exceeded")
             )
         ),
+        RecordDeletedException: create_error_handler(
+            lambda e: HTTPJSONException(
+                code=410, description=_("Record not accessible.")
```

Comment:
Minor: Since the record was deleted and anyone can access that information I would explictly say it in the message by either stating that it's gone or deleted.
```suggestion
                code=410, description=_("Record is gone.")
```

When stating not accessible it might be not fully clear and people trying to access it - without an extensive knowledge about status codes -  might think it's a permission issue.

Reply:
the problem i see is that with `Record is gone` someone also can ask `why is it gone?` there will always be a `why` question. My intention was to not reveal the real reason why it is not accessible. i had a discussion some months ago about if it is ok to reveal that a record has been deleted. the conclusion was that it should not be done, but unfortunately i forgot the justification.

Reply:
How about:
"The record associated with this file was deleted. See deletion notice" <---link to tombstone of the record

The link would be there to explain deletion reasons, since questions about this might be raised by someone, as Christoph pointed out. Not sure how difficult would be to implement the tombstone link, maybe to be done as an improvement later on, if it is not straightforward.

Reply:
i would prefer "The record associated with this file has been deleted. See deletion notice." I am not sure about the "See deletion notice", because without a link it is not that helpful, but it helps more than no indication where to find relevant information. 

my first impression is that getting a link there will be a little bit complicated. i will create an issue for that.